<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MyGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="draw_img"/>
   <addaction name="draw_rect"/>
   <addaction name="line_mode"/>
   <addaction name="clear"/>
   <addaction name="separator"/>
   <addaction name="tmp_action"/>
   <addaction name="save_action"/>
  </widget>
  <action name="draw_rect">
   <property name="text">
    <string>画矩形</string>
   </property>
   <property name="toolTip">
    <string>画矩形</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>画矩形</string>
   </property>
  </action>
  <action name="draw_img">
   <property name="text">
    <string>画图片</string>
   </property>
  </action>
  <action name="line_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线框模式</string>
   </property>
  </action>
  <action name="tmp_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/SONG/Desktop/素材/cj.ico</normaloff>C:/Users/SONG/Desktop/素材/cj.ico</iconset>
   </property>
   <property name="text">
    <string>临时按钮</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>MyGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
